### DISCUSSION TITLE
Plot assembly coverage?

### DISCUSSION THREAD

**Discussion by jaktykusuma** [Kinggerm/GetOrganelle#241] [Q&A]
Hello,

Many thanks for this amazing software. But I was wondering if you could provide (or suggest) a way to plot the assembly coverage? In the provided log file, I can only have the information on the "Average embplant_pt base-coverage = 41.2". 

Thank you for your input.

Best
 
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/10839693/get_org.log.txt)

**Comment by Kinggerm** [comment]
Yes, there is a script for doing this in GetOrganelle, `evaluate_assembly_using_mapping.py`, which can circularize the reference and generate the reads mapping plot (refer to [Fig 1](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02154-5/figures/1)). You may find the usage with `evaluate_assembly_using_mapping.py -h`.

There is [a related FAQ](https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#what-is-the-coverage-of-the-assembly) that you may want to read.
